style.css

```
@@ -0,0 +1,141 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Arial, sans-serif;
```

❗️ 수정요청
저희는 pretendard를 사용하므로 font-family에 아래처럼 명시해주셔야 해당 폰트가 적용이 됩니다~

```suggestion
  font-family: Pretendard, sans-serif;
```